source/guides/tooling/typescript-support.md

```diff
@@ -56,6 +57,6 @@ If you extend Cypress assertions, you can extend the assertion types to make the
 
 ## Additional information
 
-See the excellent advice on [setting Cypress using TypeScript](https://basarat.gitbooks.io/typescript/docs/testing/cypress.html) in [TypeScript Deep Dive](https://basarat.gitbooks.io/typescript/content/) e-book by [Basarat Syed](https://twitter.com/basarat).
+See the excellent advice on {% url "setting Cypress using TypeScript" https://basarat.gitbooks.io/typescript/docs/testing/cypress.html %} in the {% url "TypeScript Deep Dive" https://basarat.gitbooks.io/typescript/content/ %} e-book by {% url "Basarat Syed" https://twitter.com/basarat %}.
```

Comment:
yeah, I always forget to change these - since I usually copy or edit things in Markdown